lib/cli/cliUpdater.js

```diff
@@ -6,6 +6,7 @@ const { consola } = require("consola");
 
 const pkg = require("../../package.json");
 const PACKAGE_URL = "https://raw.githubusercontent.com/silverfin/silverfin-cli/main/package.json";
+const CHANGELOG_URL = "https://raw.githubusercontent.com/silverfin/silverfin-cli/main/CHANGELOG.md";
```

Comment:
This constants could be both moved to the class itself, similar to `static #UPDATE_COMMAND or `VERSION_COMMAND`